### ISSUE TITLE
Libreoffice 5.0

### ISSUE THREAD

**Issue by renearts** [mank319/elementaryPlus#297]
Libreoffice 5.0 does not seem to be recognized, I get the standard icon when elementaryPlus is selected.

**Comment by quassy** [comment]
I cannot confirm this issue with elementary+ ([1.0+242+42~ubuntu14.04.1](https://launchpad.net/~cybre/+archive/ubuntu/elementaryplus?field.series_filter=trusty)) and LibreOffice Fresh ([1:5.0.1~rc2-0ubuntu1~trusty1](https://launchpad.net/~libreoffice/+archive/ubuntu/ppa?field.series_filter=trusty)) installed from PPA. 

Are you sure elementary+ is correctly set as icon theme? You can check by typing `gsettings get org.gnome.desktop.interface icon-theme` in terminal. 

Are the elementary+ icons for LibreOffice in the correct place? You can check by typing either `find /usr/share/icons/elementaryPlus/apps/*/libreoffice*` or `~/.local/share/icons/elementaryPlus/apps/*/libreoffice*` depending on how you have installed it. (If the icons show up with one of the commands it should be fine.)

**Comment by renearts** [comment]
Both seem to be fine here. Theme is set to elementaryPlus, and the icons show up in /usr/share/icons/elementaryPlus/...
I installed it via the .deb packages, so its started with libreoffice5.0. I just removed that installation and installed it via the ppa, the icons work fine now.

**Comment by quassy** [comment]
Weird... [This `.deb` file?](hhttp://download.documentfoundation.org/libreoffice/stable/5.0.1/deb/x86_64/LibreOffice_5.0.1_Linux_x86-64_deb.tar.gz) I might try that later.

**Comment by renearts** [comment]
Yup, actually, the 5.0 variant, I didn't have 5.0.1 installed yet. But I guess it might be the same as the executable is different between .deb and the ppa (libreoffice5.0 vs. libreoffice).

**Comment by palob** [comment]
Looked into the DEB and in the ``.desktop`` launchers it uses different icon names like ``Icon=libreoffice5.0-writer`` for writer.

**Comment by renearts** [comment]
Ah, that explains a lot :)

**Comment by quassy** [comment]
We should probably at symlinks for that to be safe.
